### PR TITLE
Editorial fixes + Namespaces: distinguish vendors from IETF protocols + rephrasing "RFC series"

### LINKED ISSUE #1259: Action / Event syntax
From the review by Éric Vyncke:

## Section 1

`Action(param0, param1?, ...) / Event<param0, param1, ...>` the use of `\` is
unclear... suggest using two lines ? Should param1 in Event() also be suffixed
by a '?' ?

### LINKED ISSUE #1264: Be specific when talking about the RFC series
From the review by Éric Vyncke:

## Section 4.1

`IETF document published in the RFC Series` in which stream ? Are ISE or IAB or
IRTF streams also OK ?

### LINKED ISSUE #1268: A nit in section 8.1.11.13
From the review by Éric Vyncke:

## Section 8.1.11.13

s/It specified as the number of bytes/It *is* specified as *a* number of bytes/
?

### PR DESCRIPTION
**Pull request by mwelzl**
Closes #1259 
Closes #1268 
Closes #1264

### CODE REVIEW

**Review by gorryfair — APPROVED** [review]
This looks good

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -577,14 +578,16 @@ digits `0-9`, the hyphen `-`, and the underscore `_`. These names serve two purp
...
+  generic properties, i.e., for properties that are not specific to a protocol.
 - Protocol-specific Properties MUST use the protocol acronym as the Namespace (e.g., a
-  `tcp` Connection could support a TCP-specific Transport Property, such as the user timeout
-  value, in a Protocol-specific Property called `tcp.userTimeoutValue` (see {{tcp-uto}})).
+  `TCP` Connection could support a TCP-specific Transport Property, such as the user timeout
+  value, in a Protocol-specific Property called `TCP.userTimeoutValue` (see {{tcp-uto}})).
 - Vendor or implementation specific properties MUST use a string identifying
-  the vendor or implementation as the Namespace.
-- For IETF protocols, the name of a Protocol-specific Property SHOULD be specified in an IETF document published in the RFC Series.
+  the vendor or implementation as the Namespace. This string MUST NOT start with a capital letter,
```
Comment (philsbln) [inline draft-ietf-taps-interface.md:587]: We originally did not make any assumption about case to better blend into the language ecosystem. My mental model was always to treat them as case-insensitive. 

Using case as differentiator between vendor specific and standard attributes looks like a bad choice to me.

**Review by philsbln — CHANGES_REQUESTED** [review]
The question of using case as distinguisher needs some discussion. 

All other changes are fine and well appreticated.

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (gorryfair) [reply draft-ietf-taps-interface.md:587]: We do need a method to differentiate allocations for IETF namespaces conflicting with vendor namespaces ... can you suggest a better one? There are surely other options.

When it comes to using a property, it is possible both are simply treated the same.

... Note: all but one of the entries in the cited IETF registry that is to be used for the IETF part of the namespace are already upper case.

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -577,14 +578,16 @@ digits `0-9`, the hyphen `-`, and the underscore `_`. These names serve two purp
...
 
-- The Namespace component MUST be empty for well-known, generic properties, i.e., for
-  properties that are not specific to a protocol.
+- The optional Namespace component and its trailing character `.` MUST be omitted for well-known,
+  generic properties, i.e., for properties that are not specific to a protocol.
 - Protocol-specific Properties MUST use the protocol acronym as the Namespace (e.g., a
-  `tcp` Connection could support a TCP-specific Transport Property, such as the user timeout
-  value, in a Protocol-specific Property called `tcp.userTimeoutValue` (see {{tcp-uto}})).
+  `TCP` Connection could support a TCP-specific Transport Property, such as the user timeout
+  value, in a Protocol-specific Property called `TCP.userTimeoutValue` (see {{tcp-uto}})).
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:585]: I find this capitalization to be a bit awkward... I'd rather leave it as lowercase

**Comment by philsbln** [comment]
@mwelzl can you strip-down your PR to the editorial fixes + rephrasing "RFC series" so we can merge it right away and have the discussion on namespaces separately?

**Comment by mwelzl** [comment]
I have undone all the namespace related changes. Let's not forget that this contains a nit that we still must address: we must clarify that also the "." is optional as part of the optional namespace.

**Review by philsbln — APPROVED** [review]
Lets merge this

**Pull request merged**